core/settings.py

```
@@ -0,0 +1,654 @@
+import sys
+import warnings
+from datetime import datetime
+from pathlib import Path
+import tempfile
+import highlight_io
+from configurations import Configuration, values
+from highlight_io.integrations.django import DjangoIntegration
+from loguru import logger
+
+
+def log_warning(message:str, category, filename:str, lineno:str, file=None, line=None) ->  None:
```

Ruff (0.12.2)

Missing type annotation for function argument `category`

(ANN001)